ConsoleApp/Direction.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+    public enum Direction
+    {
+        up,
```

Comment:
code convention violated
```suggestion
        Up,
```